src/woo_publications/publications/constants.py

```diff
@@ -6,3 +6,9 @@ class PublicationStatusOptions(models.TextChoices):
     published = "gepubliceerd", _("Published")
     concept = "concept", _("Concept")
     revoked = "ingetrokken", _("Revoked")
+
+
+class DocumentActionTypeOptions(models.TextChoices):
+    signed = "ondertekening", _("Signed")
+    received = "ontvangst", _("Received")
+    declaration = "vaststelling", _("declaration")
```

Comment:
```suggestion
    declaration = "vaststelling", _("Declared")
```
to be consistent with the other two options